test/suite/wthooks.py

```diff
@@ -56,14 +56,12 @@ def tty(message):
 # Here are some useful terms to know, with some commentary for each.
 #
 #   API functions
-#      potentially any WiredTiger API functions that a hook creator wishes to modify (like
-#      Session.rename).  In Python most everything is an object.  Of course an instance of
-#      "Session" is an object, but also the "Session" class itself is an object.  The Session.rename
-#      function is also an object (of a certain form that can be called).  Also in Python,
+#      potentially any WiredTiger API functions that a hook creator wishes to modify. In Python most everything is an object.  Of course an instance of
```

Comment:
Why not change this example to `Session.remove` as well?

Reply:
I thought it was a bit repetitive but I can add it back.

Reply:
[4651212](https://github.com/wiredtiger/wiredtiger/pull/10282/commits/4651212791deac3577df5658fab5aa3c0f6f45c4)